### DISCUSSION TITLE
driver

### DISCUSSION THREAD

**Discussion by SunFloer** [seerge/g-helper#2079] [Q&A]
Can I update the driver of the ASUS keyboard here?

**Comment by seerge** [comment]
@SunFloer hello.

Updates section mirrors official Asus website with drivers for your model.

If keyboard driver is listed- i guess you can update:)

**Reply by SunFloer** [reply]
Thank youuuu，For example, is RX TKL OK?

**Reply by seerge** [reply]
@SunFloer if it's a standalone keyboard you just check on asus website directly
